packages/zosjobs/__tests__/__unit__/MonitorJobs.unit.test.ts

```diff
@@ -9,16 +9,17 @@
 *
 */
 
+jest.mock("../../src/GetJobs");
+jest.mock("@zowe/core-for-zowe-sdk");
+
 import { ImperativeError } from "@zowe/imperative";
-import { GetJobs, JOB_STATUS } from "../..";
+import { GetJobs } from "../../src/GetJobs";
+import { JOB_STATUS } from "../../src";
```

Comment:
These imports might be able to be combined.

Reply:
as long as we don't create any circular dependencies in the future 😋